src/T2DMap.cpp

```diff
@@ -4896,97 +4394,101 @@ void T2DMap::slot_setCustomLine2()
     mpCustomLinesDialog->hide(); // Hide but don't delete until done the custom line
     mCustomLinesRoomExit = exit;
     mDialogLock = false;
-    TRoom * pR = mpMap->mpRoomDB->getRoom(mCustomLinesRoomFrom);
-    if( ! pR )
+    TRoom* pR = mpMap->mpRoomDB->getRoom(mCustomLinesRoomFrom);
+    if (!pR) {
         return;
-    if( exit == "NW" )
-        mCustomLinesRoomTo = pR->getNorthwest();  // mCustomLinesRoomTo - wasn't being set!
-    else if( exit == "N" )
+    }
+    if (exit == "NW") {
+        mCustomLinesRoomTo = pR->getNorthwest(); // mCustomLinesRoomTo - wasn't being set!
+    } else if (exit == "N") {
         mCustomLinesRoomTo = pR->getNorth();
-    else if( exit == "NE" )
+    } else if (exit == "NE") {
         mCustomLinesRoomTo = pR->getNortheast();
-    else if( exit == "UP" )
+    } else if (exit == "UP") {
         mCustomLinesRoomTo = pR->getUp();
-    else if( exit == "W" )
+    } else if (exit == "W") {
         mCustomLinesRoomTo = pR->getWest();
-    else if( exit == "E" )
+    } else if (exit == "E") {
         mCustomLinesRoomTo = pR->getEast();
-    else if( exit == "DOWN" )
+    } else if (exit == "DOWN") {
         mCustomLinesRoomTo = pR->getDown();
-    else if( exit == "SW" )
+    } else if (exit == "SW") {
         mCustomLinesRoomTo = pR->getSouthwest();
-    else if( exit == "S" )
+    } else if (exit == "S") {
         mCustomLinesRoomTo = pR->getSouth();
-    else if( exit == "SE" )
+    } else if (exit == "SE") {
         mCustomLinesRoomTo = pR->getSoutheast();
-    else if( exit == "IN" )
+    } else if (exit == "IN") {
         mCustomLinesRoomTo = pR->getIn();
-    else if( exit == "OUT" )
+    } else if (exit == "OUT") {
         mCustomLinesRoomTo = pR->getOut();
-    else
-    {
+    } else {
         qWarning(R"(T2DMap::slot_setCustomLine2(): unable to identify exit "%s"to use!)", qPrintable(exit));
         return;
     }
     QList<QPointF> _list;
     pR->customLines[exit] = _list;
     QList<int> _colorList;
-//    qDebug("T2DMap::slot_setCustomLine2() NORMAL EXIT: %s", qPrintable(exit));
+    //    qDebug("T2DMap::slot_setCustomLine2() NORMAL EXIT: %s", qPrintable(exit));
     _colorList << mCurrentLineColor.red() << mCurrentLineColor.green() << mCurrentLineColor.blue();
     pR->customLinesColor[exit] = _colorList;
-/*
- *    qDebug("   COLOR(r,g,b): %i,%i,%i",
- *            mCurrentLineColor.red(),
- *            mCurrentLineColor.green(),
- *            mCurrentLineColor.blue() );
- */
+    /*
```

Comment:
Another case where (debug) code that is "carefully" commented out is being slightly mangled.  {Note: I tend to use the leading `*` on each line "C" style comment on blocks of stuff I want left there for the future (and Qt tends to put it in auto-magically as well - which means it shows up on a global search almost as well as a C++ comment.)}